src/options.ts

```diff
@@ -84,13 +90,27 @@ export function _configDefaultOptions(
             collectorEndpoint:
               process.env[Consts.OTEL_COLLECTOR_ENDPOINT] ||
               Consts.DEFAULT_COLLECTOR_ENDPOINT,
+            customHeaders: {
+              [Consts.TOKEN_HEADER_KEY]: verify_token(options.ciscoToken),
+            },
           },
         ];
 
   setInnerOptions(options);
   return <Options>options;
 }
 
+function verify_token(token: string): string {
+  if (token.startsWith('Bearer')) {
+    diag.info(
+      'Bearer was manually specified as part of the ciscoToken and recommended not to (Works either way).'
```

Comment:
maybe something like "'Bearer' prefix was attached to the provided cisco-token. We recommend using a "clean" token without the prefix"

Reply:
Fixed